src/data_tools/src/benchmark.cpp

```diff
@@ -263,6 +248,92 @@ pair<double, cv::Mat> track_error_benchmark::compute_draw_consistency_map(mbes_p
     return make_pair(meanv, error_img);
 }
 
+void track_error_benchmark::map_draw_params(PointsT& map_points, PointsT& track_points,
```

Comment:
What does this function do?

Reply:
It's equivalent to track_img_params() but it saves the parameters from the whole map to be used when drawing the heightmap of each submap independently